addon/components/au-modal.hbs

```diff
@@ -20,6 +20,7 @@
       tabindex="-1"
       {{focus-trap
         isActive=@modalOpen
+        additionalElements=(array "#ember-basic-dropdown-wormhole")
```

Comment:
If explicitly defining this doesn't seem like a good idea, this could also be a kind of a (undocumented) passthrough like: `additionalElements=@focusTrapAdditionalElements`?